tests/scripts/run_performance.sh

```diff
@@ -11,14 +11,18 @@ run_perf_models_other() {
     local tt_arch=$1
     local test_marker=$2
 
-    env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/ttnn_resnet/tests/test_perf_ttnn_resnet.py -m $test_marker
+    if [ "$tt_arch" == "grayskull" ]; then
```

Comment:
Can we please use `skip_for_grayskull` / `skip_for_wormhole_b0` and those kinds of fixtures within the test files rather than doing these if statements?

Reply:
the reason I added the if is wormhole needs to run with `WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml ` but GS does not.

Reply:
I think you can still set it to be that and not affect the runtime, correct?